IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse9.mo

```diff
@@ -165,7 +163,39 @@ First implementation for the IDEAS crash course.
 </ul>
 </html>", info="<html>
 <p>
-Adding CO2-controlled ventilation system.
+Adding CO2-controlled ventilation system. The occupancy model
+from <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4</a> is added to one zone and a 
+fixed occupancy of 1 person to the other zone. The ventilation system
+consists of two fans, two supply and two return air VAVs (Variable Air Volume), a heat recovery unit and an
+outdoor air source. The control consists of PI controllers with a set point of 1000 ppm.
```

Comment:
setpoint (maybe also apply this change to the exercise sheet of the crash course)